presto-docs/src/main/sphinx/release/release-318.rst

```diff
@@ -0,0 +1,58 @@
+===========
+Release 317
+===========
+
+General Changes
+---------------
+
+* Fix ``DISTINCT FROM`` for ``UUID`` type. (:issue:`1180`)
+* Improve performance for queries over ``information_schema`` that have a limit. (:issue:`999`)
+* Improve query performance when ``optimize_hash_generation`` is enabled. (:issue:`1071`)
+* Add support for ``Asia/Qostanay`` time zone. (:issue:`1221`)
+* Add session properties for ``query_max_memory_per_node`` and
+  ``query_max_total_memory_per_node``. (:issue:`1212`)
+* Rename Kerberos properties to make them consistent with other
```

Comment:
Which ones and how did they change?

Reply:
@sopel39?

Reply:
This needs to include the old and new property names.